Gemfile.lock

```diff
@@ -237,72 +238,72 @@ GEM
       timeout
     net-protocol (0.1.3)
       timeout
-    net-scp (3.0.0)
-      net-ssh (>= 2.6.5, < 7.0.0)
+    net-scp (4.0.0.rc1)
+      net-ssh (>= 2.6.5, < 8.0.0)
     net-smtp (0.3.1)
       digest
       net-protocol
       timeout
-    net-ssh (6.1.0)
+    net-ssh (7.0.1)
     nio4r (2.5.8)
-    nokogiri (1.13.3-x86_64-darwin)
+    nokogiri (1.13.8-x86_64-darwin)
       racc (~> 1.4)
     orm_adapter (0.5.0)
     pagy (5.10.1)
       activesupport
     parallel (1.22.1)
-    parser (3.1.1.0)
+    parser (3.1.2.1)
       ast (~> 2.4.1)
-    pg (1.3.5)
+    pg (1.4.3)
     propshaft (0.6.4)
       actionpack (>= 7.0.0)
       activesupport (>= 7.0.0)
       rack
       railties (>= 7.0.0)
-    public_suffix (4.0.6)
-    puma (5.6.4)
+    public_suffix (5.0.0)
+    puma (5.6.5)
```

Comment:
### [puma](https://puma.io)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 5.6.4 → 5.6.5

[_[change-log](https://github.com/puma/puma/blob/master/History.md), [source-code](https://github.com/puma/puma), [gem-diff](https://my.diffend.io/gems/puma/5.6.4/5.6.5)_]





<details>
<summary>Commits</summary>

A change of **18** commits. See the full changes on [the compare page](https://github.com/puma/puma/compare/v5.6.4...v5.6.5).

These are the first 10 commits:
- (316bf49) [5-6-stable branch - fix tests (#2852)](https://github.com/puma/puma/commit/316bf49d36787bec203b4f79c8df3a7aa75915a5)
- (da88536) [NullIO#closed should return false (#2883)](https://github.com/puma/puma/commit/da88536f45a7ca4b60411572cc73fbcea2daf7f5)
- (c36d644) [Fix `Puma::StateFile#load` returns empty string instead of n…](https://github.com/puma/puma/commit/c36d64405bf70bb4e9626c412a2bde37f32cdbbd)
- (3eb994e) [Do not remove pre-existing socket after USR2+TERM (#2816) (#…](https://github.com/puma/puma/commit/3eb994e87d7c9a82887e2a68b134b851215c8f75)
- (2a138c0) [Fix build with Opaque DH in LibreSSL 3.5. (#2838)](https://github.com/puma/puma/commit/2a138c0136f92e56691dcfeb267ece8a94b610cc)
- (416c1e2) [Fail hard if SSL certs or keys cannot be read by user (#2847…](https://github.com/puma/puma/commit/416c1e29f722bdb1ed428ec79b6a1581f555bacf)
- (c7dc812) [Fail hard if SSL certs or keys are invalid (#2848)](https://github.com/puma/puma/commit/c7dc812f718a2f864dca63b5b94ee04373a81697)
- (f701c4a) [Escape SSL cert and filenames (#2855)](https://github.com/puma/puma/commit/f701c4ad958a45871222af47b219c524ab8ae292)
- (e692887) [Fix rack.after_reply exceptions breaking connections (#2861)](https://github.com/puma/puma/commit/e6928872283ad2514a5044f67f550f20bd4f1a12)
- (6d22d50) [MiniSSL - detect SSL_CTX_set_dh_auto (#2864)](https://github.com/puma/puma/commit/6d22d5015b1e038bde32b8f12dc353c16127068b)


</details>